pkg/cmd/pr/checks/checks.go

```diff
@@ -14,6 +13,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const defaultInterval int = 30
```

Comment:
I think it probably would be better to drop this down to 5 or 10 seconds for the default. `run watch` uses 3 seconds but maybe checks should refresh at a slower rate.

Reply:
I set 30 seconds thinking that duration for a CI is in the order of minutes, so in the worst case is wait is 29 seconds in a multi minute run. What is your concern with this duration? I'm curious about it, in any case ok for me to reduce it. Would 10 seconds be ok?

Reply:
Exponential backoff with a maximum?

Reply:
@endorama My thought was that when trying out the new functionality 30 seconds seemed like a long time to see refreshed results. CI runs do not have to be lengthy so a shorter refresh interval would work well for both shorter runs as well as longer ones. I think 10 seconds is a good starting point, users can always customize it to their needs with the flag.

Reply:
I'll change it to 10 seconds, but I like the idea of @marekdedic about the exponential backoff (with 1 minute maximum maybe?). Is it worth the gain in terms of complexity/user experience?

Reply:
I don't think the trade off of complexity is worth it. We can always iterate on refresh interval later as we receive user feedback.

Reply:
Changed in 925c825